test/functional/inspec_vendor_test.rb

```diff
@@ -34,6 +34,25 @@
     end
   end
 
+  it 'can vendor profile dependencies from git' do
+    git_depends_path = File.join(profile_path, 'git-depends')
```

Comment:
*ahem* I think you missed a commit - I don't see a test fixture profile for this, and the functional tests fail.